packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -666,13 +682,27 @@ class CharacterActivator with Diagnosticable, MenuSerializableShortcut implement
   bool accepts(RawKeyEvent event, RawKeyboard state) {
     final Set<LogicalKeyboardKey> pressed = state.keysPressed;
     return event is RawKeyDownEvent
-      && event.character == character
+      && _characterMatches(event)
```

Comment:
Revert these changes?

Reply:
If I revert these changes, I will just close the PR, since this is the only substantive change in the PR.

The question is: are we OK with the `alt:` behavior being counterintuitive on macOS and iOS?
Using `alt:` right now is kind of redundant with the character produced on those platforms (e.g. `CharacterActivator('ß', alt: true)` is the same as `CharacterActivator('ß')`, and `CharacteActivator('s', alt: true)` will never fire).

The alternative is this code, which will use the character without modifier to allow `CharacterActivator('s', alt: true)` to fire when the non-modifier character is 's', and Option is down, making things a little less surprising. I realize that this means that the new event system will need to support an optional `characterWithoutModifier` field, but here is a use case where that is appropriate, and I don't see any other way to do the job on macOS and iOS.

If a program wants to generate its shortcuts from a string source (e.g. in the case of accelerators), and wants to know if the Alt/Option key is also down at the same time, then they have no way to implement that currently on macOS and iOS, since the option-key mapping of the keys is different for different keyboard layouts.

Luckily, for my use case I can just refuse to implement accelerators on macOS, but if someone wants to implement something that works like an accelerator they'll be unable to under the new key event system (as it is now, they could conceivably create their own activator with code like the code here, but once the old key event system is gone, they won't be able to).